.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["react", "es2015"],
+  "presets": ["react", "es2015", "stage-0"],
```

Comment:
Den er vel på stage-2. Litt skeptisk til å dra inn noe som er på et så tidlig stadie. Synes ikke at det gir så stor verdi å bruke det. Spesielt når man uansett må bruke den "vanlige syntaxen" for stateless komponenter.